gandalf.js

```diff
@@ -1,39 +1,41 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.